{
	"schema_version": "1.6.0",
	"id": "RHSA-2014:1784",
	"modified": "2024-09-15T22:06:05Z",
	"published": "2024-09-15T22:06:05Z",
	"related": [
		"CVE-2014-7144"
	],
	"summary": "Red Hat Security Advisory: python-keystoneclient security and bug fix update",
	"affected": [
		{
			"package": {
				"ecosystem": "Red Hat:openstack:5::el7",
				"name": "python-keystoneclient",
				"purl": "pkg:rpm/redhat/python-keystoneclient"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "1:0.9.0-5.el7ost"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2014:1784.json"
			}
		},
		{
			"package": {
				"ecosystem": "Red Hat:openstack:5::el7",
				"name": "python-keystoneclient-doc",
				"purl": "pkg:rpm/redhat/python-keystoneclient-doc"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "1:0.9.0-5.el7ost"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://security.access.redhat.com/data/osv/RHSA-2014:1784.json"
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY",
			"url": "https://access.redhat.com/errata/RHSA-2014:1784"
		},
		{
			"type": "ARTICLE",
			"url": "https://access.redhat.com/security/updates/classification/#moderate"
		},
		{
			"type": "ARTICLE",
			"url": "https://wiki.openstack.org/wiki/OSSN/OSSN-0024"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1106328"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1133431"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1143808"
		},
		{
			"type": "ADVISORY",
			"url": "https://access.redhat.com/security/data/csaf/v2/advisories/2014/rhsa-2014_1784.json"
		},
		{
			"type": "REPORT",
			"url": "https://access.redhat.com/security/cve/CVE-2014-7144"
		},
		{
			"type": "ADVISORY",
			"url": "https://www.cve.org/CVERecord?id=CVE-2014-7144"
		},
		{
			"type": "ADVISORY",
			"url": "https://nvd.nist.gov/vuln/detail/CVE-2014-7144"
		}
	]
}
